src/core/MOM_forcing_type.F90

```diff
@@ -802,16 +802,12 @@ subroutine extractFluxes1d(G, GV, US, fluxes, optics, nsw, j, dt, &
         endif
       endif
 
-      ! smg: we should remove sea ice melt from lprec!!!
-      ! fluxes%lprec > 0 means ocean gains mass via liquid precipitation and/or sea ice melt.
-      ! When atmosphere does not provide heat of this precipitation, the ocean assumes
+      ! When atmosphere does not provide heat of this precipitation (which here
+      ! also includes sea ice melt for legacy reasons), the ocean assumes
       ! it enters the ocean at the SST.
-      ! fluxes%lprec < 0 means ocean loses mass via sea ice formation. As we do not yet know
-      ! the layer at which this mass is removed, we cannot compute it heat content. We must
-      ! wait until MOM_diabatic_driver.F90.
       if (associated(fluxes%heat_content_lprec)) then
-        if (fluxes%lprec(i,j) > 0.0) then
```

Comment:
I believe that these changes will change answers for people (like EMC and NCAR) that were already using `fluxes%seaice_melt`.  We are probably going to need to control this with a runtime logical flag.